cluster/gce/manifests/konnectivity-server.yaml

```diff
@@ -20,7 +20,7 @@ spec:
       {{ disallow_privilege_escalation}}
       {{ capabilities }}
         {{ drop_capabilities }}
-    image: registry.k8s.io/kas-network-proxy/proxy-server:v0.31.2
+    image: registry.k8s.io/kas-network-proxy/proxy-server:v0.34.0
```

Comment:
https://github.com/kubernetes/k8s.io/pull/8719 needs to merge before this can work